docs/whats-new/dotnet-docs-mod1.md

```diff
@@ -1,102 +1,150 @@
 ---
-title: ".NET docs: What's new for July 2023"
-description: "What's new in the .NET docs for July 2023."
-ms.custom: July-2023
-ms.date: 08/01/2023
+title: ".NET docs: What's new for October 2023"
+description: "What's new in the .NET docs for October 2023."
+ms.custom: October-2023
+ms.date: 11/01/2023
 ---
 
-# .NET docs: What's new for July 2023
+# .NET docs: What's new for October 2023
 
-Welcome to what's new in the .NET docs for July 2023. This article lists some of the major changes to docs during this period.
+Welcome to what's new in the .NET docs for October 2023. This article lists some of the major changes to docs during this period.
 
 ## .NET breaking changes
 
 ### New articles
 
-- [Tool manifests in root folder](../core/compatibility/sdk/7.0/manifest-search.md)
-- [Empty keys added to dictionary by configuration binder](../core/compatibility/extensions/8.0/dictionary-configuration-binding.md)
-- [Containers default to use the 'latest' tag](../core/compatibility/sdk/8.0/default-image-tag.md)
-- [SDK no longer calls ResolvePackageDependencies](../core/compatibility/sdk/7.0/resolvepackagedependencies.md)
-- [New 'app' user in Linux images](../core/compatibility/containers/8.0/app-user.md)
-- [Default ASP.NET Core port changed to 8080](../core/compatibility/containers/8.0/aspnet-port.md)
-- [Debian container images upgraded to Debian 12](../core/compatibility/containers/8.0/debian-version.md)
-- ['libintl' package removed from Alpine images](../core/compatibility/containers/8.0/libintl-package.md)
-- [Multi-platform container tags are Linux-only](../core/compatibility/containers/8.0/multi-platform-tags.md)
+- [ManagementDateTimeConverter.ToDateTime returns a local time](../core/compatibility/extensions/8.0/dmtf-todatetime.md)
+- [--arch option doesn't imply self-contained](../core/compatibility/sdk/8.0/arch-option.md)
+
+## .NET Framework
```

Comment:
This is probably alphabetical, but might make sense to put .NET Framework near the bottom of the page since it's less important.

Reply:
Tracked as dotnet/docs-tools#271.  I'll add it to the November sprint.